crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -749,9 +901,25 @@ kitsune_p2p_types::write_codec_enum! {
         MissingOps(0x60) {
             /// The missing ops
             ops.0: Vec<(Arc<KitsuneOpHash>, Vec<u8>)>,
-            /// Is this the last chunk of ops to be sent in response
-            /// to the bloom filter that we're responding to?
-            finished.1: bool,
+            /// Ops that are missing from a bloom that you have sent.
```

Comment:
are these docs supposed to go above the `ops.0` field?

Reply:
Not really, it's more about explaining the finished `MissingOpsStatus`